### ISSUE TITLE
Ability to work with images which are large in size(Dimensions: 79007x1200)

### ISSUE THREAD

**Issue by PavelChatterjee8992** [microsoft/VoTT#1036]
I work with images that are generally large in dimensions, i.e. 79007x1200. When I add such an image, it loads, but I am unable to see the image on the screen to work with. 

The new feature could be so, that I can zoom in to a particular section or be able to see the entire image.

**Comment by mattroos** [comment]
Agreed that this zoom capability is a desirable feature. Even with "standard" 1920x1080 images, it can be hard to accurately and quickly specify bounding boxes for smaller objects. As the world moves toward 4k imagery, it will be even more difficult.
